uppsrc/Core/Inet.h

```diff
@@ -130,6 +130,8 @@ class TcpSocket : NoCopy {
 	One<SSLInfo>            sslinfo;
 	String                  cert, pkey, sni;
 	bool                    asn1;
+	
+	String                  CAcert;

```

Comment:
In U++ we do not capitalize variables. Should be changed to
`String ca_cert;`
or
`String ca_cert`